nftnl/src/expr/log.rs

```diff
@@ -0,0 +1,57 @@
+use super::{Expression, Rule};
+use nftnl_sys::{
+    self as sys,
+    libc::c_char,
+};
+
+/// A log expression.
+pub struct Log {
+    group: Option<LogGroup>,
+}
+
+#[derive(Debug, Clone, Copy, Eq, PartialEq, Hash)]
+pub enum LogGroup {
```

Comment:
I couldn't find any reason to that there's a limit of 8 groups. `u16` seems to be the [actual](https://git.netfilter.org/libnftnl/tree/src/expr/log.c) type, so I think it would make sense to use that.